simulator/autoware_auto_msgs_adapter/config/autoware_auto_msgs_adapter.param.yaml

```diff
@@ -0,0 +1,5 @@
+/**:
+  ros__parameters:
+    msg_type_target: "autoware_auto_control_msgs::msg::AckermannControlCommand"
```

Comment:
Is `msg_type_source` not required?

Reply:
I think it's OK to assume the `msg_type_source` to be the `autoware_msgs` counterpart of the message, what do you think?

I've planned this adapter to be a *one way converter* from the `autoware_msgs` to `autoware_auto_msgs` to support the simulators during their transition.

Reply:
I see. The source type is not needed if it is determined from the target type.